packages/lesswrong/server/voteServer.ts

```diff
@@ -577,6 +577,10 @@ export const recalculateDocumentScores = async (document: VoteableType, context:
     {
       documentId: document._id,
       cancelled: false
+    }, {
```

Comment:
I think a comment here would be nice - it isn't immediately obvious from context that this has any effect on the order that reactors are shown